### PR TITLE
My First Two Weeks at DockYard

### PR DESCRIPTION
**Pull request by rkrishnan8594**

### CODE REVIEW

**Comment by poteto** [comment]
I'm not sure if you're ready for this to be reviewed, but I thought it was very well written :) Great work!

File: source/posts/2016-06-17-my-first-two-weeks-at-dockyard.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Git Init: My First Two Weeks at DockYard"
+social: true
+author: Rowan Krishnan
+github: rkrishnan8594
+summary: "A brief summary of your post"
```
Comment (brzpegasus) [inline source/posts/2016-06-17-my-first-two-weeks-at-dockyard.md]: Might want to change the summary ;).

File: source/posts/2016-06-17-my-first-two-weeks-at-dockyard.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+published: true
+---
+
+# Git Init: My First Two Weeks at DockYard
```
Comment (brzpegasus) [inline source/posts/2016-06-17-my-first-two-weeks-at-dockyard.md]: You shouln't need to repeat the title here. It'll show up automatically.

File: source/posts/2016-06-17-my-first-two-weeks-at-dockyard.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+In the past couple of weeks, however, I’ve realized that there’s far more to this place than just a deep knowledge of web development.
+
+## What makes DockYard so special?
+
+One of the things that I immediately noticed clearly defines this place is the sheer amount of collective knowledge shared between everyone in the company. I’ve only just started to get to know people here, and it seems like each member of the twenty-two person team has a unique area of expertise. DockYarders include the creator of an incredible web framework, expert JavaScript and Ember.js engineers, Boston architectural enthusiasts, and craft beer sommeliers. If you put all of these people in a room together and have them work on creative projects, you’re bound to get great results. There is a contagious, inventive energy in the air that is noticeable right when you walk into the office, and you can tell that DockYarders are proud of the work that they do.
```
Comment (brzpegasus) [inline source/posts/2016-06-17-my-first-two-weeks-at-dockyard.md]: lol craft beer sommeliers. _insert @martndemus' emoji_.

**Comment by brzpegasus** [comment]
This was a fantastic post. I love your style of writing! This makes me want to work at DockYard. Oh wait, I already do! :smile:. I look forward to meeting you in person at some point.

File: source/posts/2016-06-17-my-first-two-weeks-at-dockyard.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Git Init: My First Two Weeks at DockYard"
+social: true
+author: Rowan Krishnan
+github: rkrishnan8594
+summary: "A brief summary of your post"
+published: true
+---
```
Comment (brzpegasus) [inline source/posts/2016-06-17-my-first-two-weeks-at-dockyard.md]: Perhaps add a 'culture' tag?

**Comment by rkrishnan8594** [comment]
Thanks so much @poteto and @brzpegasus! Also, thanks for picking up on some of those formatting mistakes, didn't notice those :grin:

**Comment by martndemus** [comment]
Awesome read! :shipit:

**Comment by rsocci** [comment]
Great first post @rkrishnan8594!

**Comment by hbrysiewicz** [comment]
👏  Awesome post!

File: source/posts/2016-06-17-my-first-two-weeks-at-dockyard.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+## How I Got Here
+
+Two weeks ago I started working as an intern at DockYard - though it feels like I’ve been learning from the employees here long before then. If that sounds a bit confusing, I can explain.
+
+As a young and (excessively) restless developer, I’m constantly experimenting with new libraries and frameworks. The “Projects” directory on my laptop is a messy graveyard of half-finished Todo apps, CRUD blogs, and coding sandboxes created to try out a promising new language or technology. You probably know the type, or share some of these same symptoms.
+
+![Dozens!](https://66.media.tumblr.com/8599b2611b8f19e9bb51bb629367a7ea/tumblr_o7pfsqcV6B1uk136vo1_250.gif)
+
+I decided to check out Ember.js a year ago, and soon realized that I kept finding myself back here on [ReefPoints](https://dockyard.com/blog). I was reading fantastic pieces by DockYard engineers on almost every topic pertaining to Ember development. I learned a tremendous amount about the various nooks and crannies of the framework, but more importantly, began to recognize what makes Ember such a great tool for building truly ~ambitious~ web applications.
```
Comment (maabernethy) [inline source/posts/2016-06-17-my-first-two-weeks-at-dockyard.md]: link to Ember.js website

**Pull request merged**